values.yaml

```diff
@@ -107,6 +107,12 @@ configData:
     addr: :5000
     headers:
       X-Content-Type-Options: [nosniff]
+    # Enable this if you want prometheus metrics
+    #debug:
+    #  addr: 0.0.0.0:5001
+    #  prometheus:
+    #    enabled: true
+    #    path: /metrics
```

Comment:
`configData.http.debug.prometheus` is commented-out here but is not referenced anywhere in the templates.